### ISSUE TITLE
[0.9.3 staging-1937] IndexOutOfRangeException during editing district map.

### ISSUE THREAD

**Issue by SlayksWood** [StrangeLoopGames/EcoIssues#20703]
#20194 was fixed, but I still have  another exception during editing district map.
Step to reproduce:
- connect to server, start to edit district map.
- I just change brush size and moving brush without painting (I crossed world border too).
```
IndexOutOfRangeException: Index was outside the bounds of the array.
  at Eco.Shared.Utils.Array2D`1[T].get_Item (Eco.Shared.Math.Vector2i v) [0x00000] in <00000000000000000000000000000000>:0 
  at Assets.UI.Scripts.Maps.OverlayUtils.DescribePos (Eco.Shared.Gameplay.IOverlay overlay, System.Text.StringBuilder sb, Eco.Shared.Math.Vector2i worldPos) [0x00000] in <00000000000000000000000000000000>:0 
  at MinimapUIDisplayManager.UpdateCenterText () [0x00000] in <00000000000000000000000000000000>:0 
  at MinimapUI.Update () [0x00000] in <00000000000000000000000000000000>:0 
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```
- It is not that hard to reproduce if you reconnect to the client and repeat this steps again.
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6027041/Player.log)
[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/6027042/Player-prev.log)
Video: https://drive.google.com/file/d/1q6EHeTPlU7JkVKfT6N7vAI6xFo23rOgN/view?usp=sharing

**Comment by SlayksWood** [comment]
Still have this exception. But I investigate it more and seems I pointed out the wrong problem(or original was fixed and this is just another case). When you did the steps I described in the original problem, you did what caused the exception.
So step to reproduce:
- start to draft district map.
- Scroll the map to zero point:
![image](https://user-images.githubusercontent.com/45708377/110900805-67dc2f80-8314-11eb-83c7-eff33f8dd3cd.png)
- Divide the screen into 4 parts and move the zero point to different parts (I tried to do this in a circle):
![image](https://user-images.githubusercontent.com/45708377/110901126-e0db8700-8314-11eb-96ec-8ab1198f93ff.png)
Video: https://drive.google.com/file/d/1uBJS8q1WF9TXprS77RjbxhNvxTSR7hqS/view?usp=sharing
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/6128010/Player.log)
